Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts

```diff
@@ -99,6 +100,14 @@ async function run() {
         if(webAppUri) {
             tl.setVariable(webAppUri, publishingProfile.destinationAppUrl);
         }
+		
+		if(renameFilesFlag){
+			var obj = [];
+			await azureRESTUtility.getWebAppAppSettings(SPN, webAppName, resourceGroupName, deployToSlotFlag, slotName, obj);
+			obj[0].properties.MSDEPLOY_RENAME_LOCKED_FILES='1';
```

Comment:
Continuation of previous comment... not sure if you can blindly set '1'. You should handle disable of this flag as well.